client/app/bundles/SearchPage/components/searchTab/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDom from 'react-dom';
+
+import DropDown from './dropdown';
+import CategoryTab from './categoryTab';
+
+import _ from 'lodash';
+
+export default class SearchTab extends Component {
+  static propTypes = {
+    // categoryStats: PropTypes.object.isRequired,
+    category_stats: PropTypes.arrayOf(
+                      PropTypes.shape({
+                        category: PropTypes.string.isRequired,
+                        count: PropTypes.string.isRequired
+                      })
+                    ).isRequired,
+
+    active_tab: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired
+  }
+
+  constructor() {
```

Comment:
Why do we have a constructor just to call super? Surely this gets called anyway?